src/components/Button/Button.js

```diff
@@ -60,12 +61,25 @@ const REGULAR_BUTTON_ONLY_PROPS = [
  * A button component with support for the anchor and button
  * element as well as a button-looking button and a text link.
  */
-const Button = ({ plain, ...props }) =>
-  plain ? (
-    <PlainButton {...omit(REGULAR_BUTTON_ONLY_PROPS, props)} />
+const Button = ({ plain, trackingLabel, onClick, ...props }) => {
+  const dispatch = useClickTrigger();
+  const handler = e => {
+    dispatch({ label: trackingLabel, component: 'button' });
```

Comment:
We also need to add a prop to receive customParameters (previously called the data object)

Reply:
If we use label to decide if a tracking event will be sent or not, that means label is a necessary data for tracking event. This doesn't match with what we mark in collector. Inside collector, label is an optional value. We could change collector of course. My other thought is that in marketing website, label is set by editors, so it is also an optional value. For some components, we also send out tracking event without this value. But I think this could be discussed component by component too. 

sorry I forget that one. Will add it today.